lib/htmlnano.es6

```diff
@@ -62,6 +81,12 @@ function htmlnano(optionsRun, presetRun) {
     };
 }
 
+htmlnano.getRequiredOptionalDependencies = function (optionsRun, presetRun) {
+    const [options] = loadConfig(optionsRun, presetRun);
+
+    return [...new Set(Object.keys(options).filter(moduleName => options[moduleName]).map(moduleName => optionalDependencies[moduleName]).flat())];
+};
```

Comment:
`getRequiredOptionalDependencies` API is designed for other libraries that utilize htmlnano.

For example, parcel-bundler has a plugin `@parcel/optimizer-htmlnano` built on top of `htmlnano`. With this API, the parcel can auto-install those optional dependencies.